optimum_benchmark/benchmarks/energy_star/preprocessing_utils.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from datasets import Dataset
+from transformers import AutoTokenizer
+
+from .config import EnergyStarConfig
+
+
+def preprocess(dataset: Dataset, task: str, config: EnergyStarConfig, model_name: str) -> Dataset:
+    task_to_preprocessing = {"feature-extraction": feature_extraction_preprocessing}
+
+    return task_to_preprocessing[task](dataset, config, model_name)
+
+
+def feature_extraction_preprocessing(dataset: Dataset, config: EnergyStarConfig, model_name: str) -> Dataset:
```

Comment:
`feature-extraction`, if we stay aligned with Transformers' pipelines, only deals with text. There is an [`image-feature-extraction`](https://github.com/huggingface/transformers/blob/7e1413d16ab34dbfbdb0019735f7fa33a57c658f/src/transformers/pipelines/__init__.py#L364) pipeline for images.
So my idea was to make the task name appear in the preprocessing method name so that it is obvious which task it is used for.